### DISCUSSION TITLE
All modes are sorted by the standard mode's difficulty.

### DISCUSSION THREAD

**Discussion by Qnene** [ppy/osu#22696] [Ideas] [answered]
Beatmaps in all modes are sorted by the standard mode's difficulty
Converted bitmaps should be sorted by converted difficulty.

The difficulty range slider has the same problem.

**It's very inconvenient, I made a Github account because of this problem**

**Comment by bdach** [comment, marked as answer]
Duplicate of https://github.com/ppy/osu/issues/12628
